indico/modules/events/abstracts/controllers/display.py

```diff
@@ -90,8 +90,8 @@ def _process(self):
 class RHSubmitAbstract(RHDisplayAbstractsBase):
     def _process(self):
         if not self.event_new.cfa.can_submit_abstracts(session.user):
-            return WPSubmitAbstract.render_template('display/submit_abstract.html', self._conf, event=self.event_new,
-                                                    form=None)
+            return redirect(url_for('event.conferenceCFA', self.event_new))
```

Comment:
@mvidalgarcia, that's what @ThiefMaster means. Maybe this is the moment to stop using legacy 😉